### ISSUE TITLE
Dependencies Debian

### ISSUE THREAD

**Issue by EtienneBruines** [paked/engi#7]
As mentioned at their GitHub page, [glfw](https://github.com/go-gl/glfw) has some dependencies. 

For example, on Debian/Ubuntu, you need to:
```Shell
sudo apt-get install mesa-common-dev xorg-dev libgl1-mesa-dev
``` 

These are the only dependencies I've found (so far). I'll leave it up to you if you want to document this, and how you want to document this.

**Comment by paked** [comment]
These definitely need to be part of the README file. Silly oversight by me.

**Comment by paked** [comment]
Hang on, these are build-only dependencies right? Or are the required *whenever* running an engi game?

**Comment by EtienneBruines** [comment]
Have not noticed this comment before. Apologies. 

Since they are `-dev` packages, I assume they are build-only dependencies. I'm not entirely sure as I do not have a system running without those packages; so I won't be able to say for sure.

**Comment by EtienneBruines** [comment]
Fixed as per #88.
